src/devtools/client/debugger/src/components/Editor/Tabs.tsx

```diff
@@ -35,12 +35,13 @@ class Tabs extends PureComponent<PropsFromRedux> {
   _draggedSource: SourceDetails | { url: null; id: null } | null = null;
   _draggedSourceIndex: number | null = null;
 
+  sourceTabs = React.createRef<HTMLDivElement>();
```

Comment:
Subjective naming nit (please feel free to ignore) – I prefer to prepend refs with "ref" (e.g. `sourceTabRef`) so it's clear at a glance that they are a wrapper and not the thing itself.

Reply:
Yeah, agreed. I think I was keeping to the existing naming because it was already in `this.refs`.